src/category_theory/preadditive/scratch2.lean

```diff
@@ -0,0 +1,23 @@
+import category_theory.limits.shapes.biproducts
```

Comment:
I don't think you intended to PR this file. Please remove it from your branch :wink:

Reply:
Done.